install/kubernetes/helm/istio/charts/security/templates/meshexpansion.yaml

```diff
@@ -53,4 +53,27 @@ spec:
           number: 8060
 ---
 {{- end }}
+
+{{- if .Values.global.controlPlaneSecurityEnabled }}
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: meshexpansion-dr-citadel
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "security.name" . }}
+    chart: {{ template "security.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    istio: citadel
+spec:
+  host: istio-citadel.{{ .Release.Namespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
+  trafficPolicy:
+    portLevelSettings:
+    - port:
+        number: 8060
+      tls:
+        mode: DISABLE
```

Comment:
Does this mean we disabled mTLS for citadel at all? Is there a way to make mTLS works for citadel?

Reply:
The funny thing is that the same DR is set for pilot -> https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/pilot/templates/meshexpansion.yaml#L69-L87